xml/System.Net/WebClient.xml

```diff
@@ -1612,18 +1612,15 @@
  For more information about how to handle events, see [Handling and Raising Events](/dotnet/standard/events/).  
   
 > [!NOTE]
-> A passive FTP file transfer will always show a progress percentage of zero, since the server did not send the file size. To show progress, you can change the FTP connection to active by overriding the <xref:System.Net.WebClient.GetWebRequest%2A> virtual method:  
+> A passive FTP file transfer will always show a progress percentage of zero, since the server did not send the file size. To show progress, you can change the FTP connection to active by overriding the <xref:System.Web.Services.Protocols.WebClientProtocol.GetWebRequest(System.Uri)> virtual method:  
```

Comment:
@scottaddie The WebClient.GetWebRequest method doesn't seem to exist anymore, so I replaced the link with WebClientProtocol.GetWebRequest. Is that correct, and is the example that follows still applicable?